examples/cms-dotcms/components/blocks.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
```

Comment:
Importing react if unused is not necessary anymore.

```suggestion
```

Reply:
Done.